sklearn/neighbors/_nearest_centroid.py

```diff
@@ -94,6 +97,11 @@ class NearestCentroid(ClassifierMixin, BaseEstimator):
     [1]
     """
 
+    _parameter_constraints = {
+        "metric": StrOptions({"euclidean", "manhattan"}),
```

Comment:
This needs to always be a list.

```suggestion
        "metric": [StrOptions({"euclidean", "manhattan"})],
```

Reply:
Alright thanks, that was my bad in the original suggestion.